From 5874a0813cf843094cbe422cf9f07964e0e6199e Mon Sep 17 00:00:00 2001
From: chenxia1 <xiao.x.chen@intel.com>
Date: Fri, 26 Jul 2019 09:20:34 +0800
Subject: [PATCH] phonedoctor: set crashtool server address

set server address for phonedoctor to upload the events
set the intel certificate

Change-Id: Ia7a4c6ba2758b679b1da74be1b14357861c34c60
Tracked-On:
Signed-off-by: chenxia1 <xiao.x.chen@intel.com>
---
 crashreport/res/raw/crashtool.crt             | 23 +++++++++++++++----
 crashreport/res/values/strings.xml            |  2 +-
 .../intel/crashreport/core/GeneralEvent.java  |  2 +-
 .../crashtoolserver/bean/Event.java           |  2 +-
 4 files changed, 22 insertions(+), 7 deletions(-)

diff --git a/crashreport/res/raw/crashtool.crt b/crashreport/res/raw/crashtool.crt
index b6f6ef0..b216612 100755
--- a/crashreport/res/raw/crashtool.crt
+++ b/crashreport/res/raw/crashtool.crt
@@ -1,7 +1,22 @@
 Bag Attributes
-    friendlyName:
-    localKeyID:
-subject=
-issuer=
+    friendlyName: clota
+    localKeyID: 54 69 6D 65 20 31 35 33 37 37 37 37 35 35 36 34 31 35 
+subject=/C=In/ST=Karnataka/L=Bangalore/O=intel/OU=ssg-otc/CN=clota.intel.com
+issuer=/C=In/ST=Karnataka/L=Bangalore/O=intel/OU=ssg-otc/CN=clota.intel.com
 -----BEGIN CERTIFICATE-----
+MIICsDCCAhmgAwIBAgIJAM+W0EpMcLXUMA0GCSqGSIb3DQEBCwUAMHExCzAJBgNV
+BAYTAkluMRIwEAYDVQQIDAlLYXJuYXRha2ExEjAQBgNVBAcMCUJhbmdhbG9yZTEO
+MAwGA1UECgwFaW50ZWwxEDAOBgNVBAsMB3NzZy1vdGMxGDAWBgNVBAMMD2Nsb3Rh
+LmludGVsLmNvbTAeFw0xODA5MjMwNDU3MjFaFw0yMTA3MTMwNDU3MjFaMHExCzAJ
+BgNVBAYTAkluMRIwEAYDVQQIDAlLYXJuYXRha2ExEjAQBgNVBAcMCUJhbmdhbG9y
+ZTEOMAwGA1UECgwFaW50ZWwxEDAOBgNVBAsMB3NzZy1vdGMxGDAWBgNVBAMMD2Ns
+b3RhLmludGVsLmNvbTCBnzANBgkqhkiG9w0BAQEFAAOBjQAwgYkCgYEAvIG7o2lZ
+Eh0s6Fhe8mO5S2k2GSjaCP12SRuqUmvnx6jKhEFdmf2UhO1aEXu6LlZZbOlsTJyL
+lmRLvR1Lbg9E9RAf9hWD8OOc0j5/pAt3WuDBfSry46OWLFx+8qcn3X7Zl0Jr1U0G
+ydyR364uTDni6RZyEPStfQzvjCcNNINapTMCAwEAAaNQME4wHQYDVR0OBBYEFClg
+w8xF+cuJEv3cLUvUL8S0RJIyMB8GA1UdIwQYMBaAFClgw8xF+cuJEv3cLUvUL8S0
+RJIyMAwGA1UdEwQFMAMBAf8wDQYJKoZIhvcNAQELBQADgYEALCs9U/OpvCM5gM0T
+IBXKeCWCxfKj04YCPSm0FDSPXhPuqoyYMAePyJX80gvkMmxQjmV+QQKuMdCiCgpc
+2e+krnczOn7e0lfD9KYzLq6gCeZNJagoR6ca+1Icr1JZJmr8223/pKxCa55eWbi8
+Sc/CyQu0gYCSqqxfJXtZId4rZOQ=
 -----END CERTIFICATE-----
diff --git a/crashreport/res/values/strings.xml b/crashreport/res/values/strings.xml
index b9b9aeb..a02db21 100755
--- a/crashreport/res/values/strings.xml
+++ b/crashreport/res/values/strings.xml
@@ -24,7 +24,7 @@
 
     <!-- Server address -->
     <string name="settings_private_app_server_host_key">settings_private_app_server_host</string>
-    <string name="settings_private_app_server_host_value">server</string>
+    <string name="settings_private_app_server_host_value">clota.intel.com</string>
 
     <!-- Server port -->
     <string name="settings_private_app_server_port_key">settings_private_app_server_port</string>
diff --git a/libpd-core/src/com/intel/crashreport/core/GeneralEvent.java b/libpd-core/src/com/intel/crashreport/core/GeneralEvent.java
index 64ebf76..cf71f82 100755
--- a/libpd-core/src/com/intel/crashreport/core/GeneralEvent.java
+++ b/libpd-core/src/com/intel/crashreport/core/GeneralEvent.java
@@ -497,7 +497,7 @@ public class GeneralEvent {
 				getType(), getData0(), getData1(), getData2(), getData3(),
 				getData4(), getData5(), date, convertUptime(getUptime()),
 				null /*logfile*/, sBuild,
-				com.intel.crashtoolserver.bean.Event.Origin.NULL, device,
+				com.intel.crashtoolserver.bean.Event.Origin.CLOTA, device,
 				getiRowID(), pdStatus, this.osBootMode,
 				new com.intel.crashtoolserver.bean.Modem(getModemVersionUsed()),
 				getCrashDir(), uploaded, logUploaded, tc);
diff --git a/libpd-crashtool/crashtoolserver/bean/Event.java b/libpd-crashtool/crashtoolserver/bean/Event.java
index d55d9c1..d77f39c 100755
--- a/libpd-crashtool/crashtoolserver/bean/Event.java
+++ b/libpd-crashtool/crashtoolserver/bean/Event.java
@@ -40,7 +40,7 @@ public class Event implements Serializable {
 	 * Origin exhaustive list
 	 * default value : GENE
 	 */
-	public enum Origin {NULL}
+	public enum Origin {ACS, CLA, CLOTA, EGG, GENE, MPM, MPTA, PDN, PFT, SRC, STUB}
 
 	public static final int UPLOADED_STATUS_INVALID = -1;
 	public static final int UPLOADED_STATUS_NOT_UPLOADED = 0;
-- 
2.20.1

